### DISCUSSION TITLE
How can I execute an AppleScript on close?

### DISCUSSION THREAD

**Discussion by sdelamo** [ttscoff/bunch-docs#128] [Q&A] [answered]
The following snippet at the end of my bunch works:

```
...
!<<#On Close
___
#[On Close]
* ~/Library/Scripts/Stop\ Harvest.scpt
```

However, Is there a more succinct way of executing the apple script on close?

**Comment by ttscoff** [comment, marked as answer]
This should work:

!* script…

- Brett
